TreeMaker/python/triggerNameList.py

```diff
@@ -8,6 +8,10 @@
 triggerNameList = cms.vstring(
     # handle duplicates
 	list(sorted(set([
+    'DST_DoubleMu3_noVtx_CaloScouting_v',

```

Comment:
is it worthwhile to have scouting triggers in this list? do they show up in regular (non-scouting) data or MC?

Reply:
I'm not sure. It's something I want to discuss at today's meeting. I've made a slide about this to gather some input. My gut feeling is no and that these should be removed, but as they were sent to me as possible interesting triggers, I figured I should include them to start with.

Reply:
It appears that the scouting triggers are not saved in our signal samples nor in the background samples. In my tests, I ran over 5000 signal events (Fall17 flavored model) and 25234 TTJets_SingleLeptFromT events (Fall17). I found that the DST triggers never passed (never was there a 1 in the `TriggerPass` branch). For the displaced triggers I found that at least on of them fired for 1880 events in signal and 185 events in background. So I know these are being simulated, even if they are <50% efficient from the start.

Commands used:
```
PreSelection->GetEntries()
((TBranch*)PreSelection->GetBranch("TriggerPass"))->GetTitle()
PreSelection->Scan("TriggerPass[0]:TriggerPass[1]:TriggerPass[2]:TriggerPass[3]","TriggerPass[0]==1 || TriggerPass[1]==1 || TriggerPass[2] == 1 || TriggerPass[3] == 1")
PreSelection->Draw("TriggerPass[48]==1 || TriggerPass[49]==1 || TriggerPass[50]==1 || TriggerPass[51]==1 || TriggerPass[52]==1 || TriggerPass[53]==1 || TriggerPass[54]==1 || TriggerPass[55]==1 || TriggerPass[56]==1 || TriggerPass[57]==1 || TriggerPass[58]==1 || TriggerPass[59]==1 || TriggerPass[60]==1 || TriggerPass[61]==1 || TriggerPass[62]==1 || TriggerPass[63]==1 || TriggerPass[64]==1 || TriggerPass[65]==1 || TriggerPass[66]==1","")
```

Reply:
Can you check if the DST triggers have TriggerPass values of 0 or -1? The latter would indicate the trigger wasn't found in the list for that event at all (rather than being found and failing).

Reply:
Using [1] I find that all 5000 or 25234 events have `TriggerPass==-1`. So these triggers aren't even being found.


[1]
```
PreSelection->Draw("TriggerPass[0]==0 || TriggerPass[1]==0 || TriggerPass[2]==0 || TriggerPass[3]==0","TriggerPass[0]!=1 || TriggerPass[1]!=1 || TriggerPass[2] != 1 || TriggerPass[3] != 1")
PreSelection->Draw("TriggerPass[0]==-1","TriggerPass[0]!=1")
PreSelection->Draw("TriggerPass[1]==-1","TriggerPass[1]!=1")
PreSelection->Draw("TriggerPass[2]==-1","TriggerPass[2]!=1")
PreSelection->Draw("TriggerPass[3]==-1","TriggerPass[3]!=1")
```